File: tests/lua/unitTests/featureFlag.lua
local ServerStorage = game:GetService("ServerStorage")

local DatalinkService = require(ServerStorage.DatalinkService)

local FAST_FLAGS = {
	"ExampleFlag"
}

local fastFlagTest = { }

function fastFlagTest.run()
	for _, flagName in FAST_FLAGS do
		print(flagName, DatalinkService:GetFastFlag(flagName))
	end

	return true
end

return fastFlagTest

File: src/tests/ts/unitTests/featureFlag.ts
import DatalinkService from '../../..'

const FAST_FLAGS = [
    "ExampleFlag"
]

class fastFlagTest {
    run() {
        FAST_FLAGS.forEach((flagname: string)=> {
            print(`Fetching ${flagname}`, DatalinkService.GetFastFlag(flagname))
        })
    }
}

export { fastFlagTest }